configurator/src/store/modules/keycodes/app-media-mouse.js

```diff
@@ -42,49 +41,21 @@ export default [
 
   { label: 'Mouse Keys', width: 'label' },
 
-  { name: 'Mouse Up', code: 'KC_MS_U', title: 'KC_MS_U - Mouse Cursor Up' },
-  {
-    name: 'Mouse Down',
-    code: 'KC_MS_D',
-    title: 'KC_MS_D - Mouse Cursor Down'
-  },
-  {
-    name: 'Mouse Left',
-    code: 'KC_MS_L',
-    title: 'KC_MS_L - Mouse Cursor Left'
-  },
-  {
-    name: 'Mouse Right',
-    code: 'KC_MS_R',
-    title: 'KC_MS_R - Mouse Cursor Right'
-  },
+  { name: 'Mouse Up', code: 'KC_MS_U', title: 'Mouse Cursor Up' },
+  { name: 'Mouse Down', code: 'KC_MS_D', title: 'Mouse Cursor Down' },
+  { name: 'Mouse Left', code: 'KC_MS_L', title: 'Mouse Cursor Left' },
+  { name: 'Mouse Right', code: 'KC_MS_R', title: 'Mouse Cursor Right' },
   { width: 250 },
```

Comment:
Not sure these should change. It's either or not both. What do you think?

Reply:
I'm not sure I understand what you're getting at.

Reply:
Have you tested it? You should see the issue

Reply:
I mean, it shows a tooltip of both the keycode and the title on the tabs, and the keycode only if it's in the keymap (which I don't remember seeing until today). Is that what you're getting at?

Reply:
I'm kind of surprised. It's either or. Either show the code or the title. I think it's a good idea to show both though and I may make that change.